src/module.ts

```diff
@@ -511,8 +513,50 @@ export default defineNuxtModule<ModuleOptions>({
       ...contentContext as any
     }
 
+    // @nuxtjs/tailwindcss support
+    // @ts-ignore - Module might not exist
+    nuxt.hook('tailwindcss:config', (tailwindConfig) => {
+      tailwindConfig.content = tailwindConfig.content ?? []
+      tailwindConfig.content.push(`${nuxt.options.buildDir}/cache/content/parsed/**/*.md`)
+    })
+
     // Setup content dev module
-    if (!nuxt.options.dev) { return }
+    if (!nuxt.options.dev) {
+      nuxt.hook('build:before', async () => {
+        const storage = createStorage()
+        const sources = useContentMounts(nuxt, contentContext.sources)
+        sources.cache = {
+          driver: 'fs',
+          base: resolve(nuxt.options.buildDir, 'cache')
```

Comment:
Please use module prefix for custom cache dir (can be `.nuxt/content-cache`)

Reply:
This is not a custom cache, this code tries to initialize and fill Nuxt cache. Having another directory for the cache will result duplicating contents and tasks

Reply:
You can mount `/cache/content` to `/cache` this way you can both predict real fs path and also any write to `/cache/content` goes there not being affected by nuxt and nitro defaults.

Reply:
> This implies double fetching remote sources even in dev mode, which had performance downsides for users in dev mode.

@farnabaz Why do you think it cases double fetching? By default keys to `/cache/*` are mapped to `.nuxt/cache/*`. By mounting, it remains same and any write to `/cache/content/*` maps to `/.nuxt/content-cache/*` instead).

Reply:
Right now we are fetching remote sources twice, once in `before:build` hook and another in nitro runtime. But this only happens in build time. If we change tailwind search dir, we need to fetch contents in development mode too. And have duplicate fetch in dev mode.

Reply:
Fetching would be the same also for dev as unstorage keys remain the same as before. Maybe i can explain better this way, currently we assume during build and dev, a key write to `cache:content:xyz` maps to `.nuxt/cache/content/xyz` file. This might change in the future. Nuxt might use `.nuxt/cache/[dev|prod]/content/xyz` or even `.nuxt/cache/db.sql`. 

If you (dev-build and prod-build) mount a nitro storage to `/cache/content` and explicitly mapping to single fs path `.nuxt/content-cache`, you can be sure a write to `cache:content:xyz` goes to certain directory structure expected by tailwind or any other internal wants to interact with cache without storage layer but directly accessing it.

With what i suggested, both tailwind and content, still write to **one path** but a predictable one that you expect :)

This change is probably not directly related to this PR but since i spotted a hard code dependency on fs path, noted this to be solved sooner and avoid breaking content.

Reply:
Are you suggesting adding `/cache/content` mount to nitro storage?

Reply:
@pi0  by `.nuxt/content-cache`, you mean `${nuxt.option.rootDir}/.nuxt` ? or keep using `buildDir`?

Reply:
Yes. I use `.nuxt` instead of `buildDir` in examples to be readable :)